org-roam.el

```diff
@@ -532,8 +532,8 @@ PATH should be the root from which to compute the relativity."
       ;; Loop over links
       (while (re-search-forward org-roam--org-link-bracket-typed-re (point-max) t)
         (setq link (match-string 2))
-        (when (and (file-attributes link)
-                   (f-relative-p link))
+        (when (and (f-relative-p link)
+                   (file-attributes link))
```

Comment:
There's no need for the `file-attributes` check anymore, since we are no longer using it to determine whether the path is a URL.

Reply:
> we are no longer using it to determine whether the path is a URL.
Is this reflected in this fix/windows branch, or it's in the master?

Are you suggesting that we should simply remove the `and` condition like this?
`(when (f-relative-p link) ... [omitted] ... )`

Reply:
> Are you suggesting that we should simply remove the and condition like this?
(when (f-relative-p link) ... [omitted] ... )

yes. I added it in the `fix/windows` branch, because I thought it would be a fast way to not do processing on non file links.